Resources/public/inlineDeleteCheckboxes.js

```diff
@@ -0,0 +1,32 @@
+/*
+
+ This file is part of the Sonata package.
+
+ (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+ */
+
+var handleInlineDeleteCheckboxes = function() {
+    var checkboxes = jQuery('.sonata-admin-type-delete-checkbox');
+
+    if(checkboxes.length) {
+        var eventType = window.SONATA_CONFIG.USE_ICHECK ? 'ifChanged': 'change';
+
+        checkboxes.on(eventType, function() {
```

Comment:
`ifchanged, change` these are the events to handle the checkbox being checked
`ready sonata-admin-setup-list-modal sonata-admin-append-form-element sonata.add_element` these ones are to reinitialize the listener to the above events on dynamically added content .Don't really see how i can mix those